src/pages/ShopDetail/ShopDetail.js

```diff
@@ -1,10 +1,189 @@
 import React from 'react';
+import { REVIEW_URL } from '../../config';
+import RestaurantReview from './RestaurantReview/RestaurantReview';
+import RestaurantInfo from './RestaurantInfo/RestaurantInfo';
+import './ShopDetail.scss';
 
 class ShopDetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      wanted: false,
+      data: [],
+      updateReviewdata: [],
+      quantity: 9,
+      moreButtonHidden: false,
+    };
+  }
+  goToWrtingPage = () => {
+    this.props.history.push('/shopdetail-reviewwritingpage');
+  };
+
+  //   handleWanted = () => {
+  //     let update = [...this.state.data];
+  //     update.
+  // this.setState({
+
+  // });
+  //    is_wished
+  //   };
+
+  handleDelete = response => {
+    let updateReviewdata = [];
+    for (let i = 0; i <= 4; i++) {
+      if (response.results[0].review[i]) {
+        updateReviewdata = updateReviewdata.concat(
+          response.results[0].review[i]
+        );
+      }
+    }
+
+    this.setState({
+      data: response.results[0],
+      updateReviewdata: [...updateReviewdata],
+    });
+  };
+
+  moreButton = () => {
+    const { data, quantity } = this.state;
+    let updateReviewdata = [];
+    if (data.review.length > quantity) {
+      for (let i = 0; i <= quantity; i++) {
+        updateReviewdata = updateReviewdata.concat(data.review[i]);
+      }
+      this.setState({
+        updateReviewdata: [...updateReviewdata],
+        quantity: quantity + 5,
+      });
+    } else if (4 > quantity - data.review.length > 0) {
+      for (let i = quantity - 4; i < data.review.length; i++) {
+        updateReviewdata = updateReviewdata.concat(data.review[i]);
+      }
+      let remain = [...this.state.updateReviewdata];
+      remain = remain.concat(updateReviewdata);
+      this.setState({
+        updateReviewdata: [...remain],
+        quantity: quantity + 4,
+        moreButtonHidden: true,
+      });
+    }
+  };
+  // mock data
+  componentDidMount() {
+    fetch('http://localhost:3000/data/restaurantdetail.json')
```

Comment:
`http://localhost:3000` 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에, 수정해주시기 바랍니다.
```suggestion
    fetch('/data/restaurantdetail.json')
```

Reply:
수정했습니다 !